rest/data/commitq.go

```diff
@@ -0,0 +1,37 @@
+package data
+
+import (
+	"github.com/evergreen-ci/evergreen/model/commitq"
+	"github.com/pkg/errors"
+)
+
+type DBCommitQConnector struct{}
+
+func (pc *DBCommitQConnector) EnqueueItem(queueId string, item string) error {
+	q, err := commitq.FindOneId(queueId)
+	if err != nil {
+		return errors.Wrapf(err, "can't query for queue id %s", queueId)
+	}
+	if q == nil {
+		return errors.Errorf("commit queue not found for id %s", queueId)
+	}
+
+	if err := q.Enqueue(item); err != nil {
```

Comment:
Right now this is checked by the the github rest route that does the enqueuing.
I could see putting the check here as well since there will be other methods of adding to the queue.

Reply:
I moved things around a bit but now it's checked here (in the call to FindOneProjectRefWithCommitQByOwnerRepoAndBranch we make sure the queue is enabled)